src/main/java/com/rakovets/course/java/core/practice/jcf/collection/Dictionary.java

```diff
@@ -0,0 +1,141 @@
+package com.rakovets.course.java.core.practice.jcf.collection;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Dictionary {
+    private Map<String,String> dictionary;
+    private Map<String, Integer> counterTop;
+
+    public Dictionary(Map<String, String> dictionary, Map<String, Integer> counterTop) {
+        this.dictionary = dictionary;
+        this.counterTop = counterTop;
+    }
+
+    public Dictionary(Map<String, String> dictionary, Map<String, Integer> counterTop, String firstWord, String firstTranslate) {
+        this.counterTop = counterTop;
+        this.dictionary = dictionary;
+        dictionary.put(firstWord, firstTranslate);
+    }
+
+    public String putNewWordAndTranslate(String word, String translate) {
+        if (dictionary.containsKey(word)) {
+            return word + ": this word already in dictionary. You can add/delete translate if you want";
+        } else {
+            dictionary.put(word, translate);
+            return word + ": added in dictionary";
+        }
+    }
```

Comment:
This API is unnatural:
```suggestion
    public boolean putNewWordAndTranslate(String word, String translate) {
        if (dictionary.containsKey(word)) {
            return false;
        } else {
            dictionary.put(word, translate);
            return true;
        }
    }
```